modules/jupyter/variables.tf

```diff
@@ -135,3 +135,9 @@ variable "ephemeral_storage" {
 variable "autopilot_cluster" {
   type = bool
 }
+
+variable "rag" {
```

Comment:
Meaningful name, and update the description to explain why and when we use a custom image

Reply:
Changed to "use_custome_image"